src/smol/smol-slime-alternative-parts-guide.md

```diff
@@ -0,0 +1,181 @@
+    </tr>
+    <tr>
+        <td>eByte Dongle (E104-BT5040U)</td>
+        <td>Cheapest option with a PCB antenna.<br/> 
+        - <strong>E104-BT5040U</strong> is model that you need. It is fully compatible with the Nordic Semiconductor nRF52840 Dongle.<br/>
+        - <strong>E104-BT5040UA</strong> is not compatible. It is only capable of capturing BLE4.2 and BLE5.0 protocol packets.</td>
+        <td>It is available on AliExpress, with free shipping.<br/>
+        Also available on Alibaba, where sold by <strong>Chengdu Ebyte Electronic Technology Co., Ltd.</strong></td>
+    </tr>
+    <tr>
+        <td>Nordic Semiconductor nRF52840 Dongle (PCA10059)</td>
+        <td>Official Nordic development hardware.</td>
+        <td>It is available on <strong>Digikey</strong> or <strong>Mouser</strong>.</td>
+    </tr>
+</table>
+
+### Microcontrollers Modified Into USB Dongles
+If you want to improve signal strength, you can replace the built-in antenna with a 31 mm wire. This creates a basic monopole antenna.
+
+<table>
+    <tr>
+        <th>Board</th>
+        <th>Description</th>
+        <th>Obtaining</th>
+    </tr>
+    <tr>
+        <td>SuperMini nRF52840</td>
+        <td>It is a clone of the <strong>nice!nano</strong> board.</td>
+        <td>Cheapest option overall.<br/>
+         It is available on AliExpress with <code>compatible with nice!nano</code> or <code>Pro Micro</code> branding.</td>
+    </tr>
+    <tr>
+        <td>Seeed Studio XIAO nRF52840</td>
+        <td>Compact board.</td>
+        <td>Available on <strong>Seeed Studio</strong>.</td>
+    </tr>
+</table>
+
+## 🏃 Trackers
+Before you start, decide on [how many trackers you may need](../slimevr101.md#how-many-trackers-do-you-need).
+
+Trackers are required to have a battery and an inertial measurement unit (IMU). A magnetometer is optional.\
+Buttons and slide switches are recommended but not required. Buttons can be added to control the tracker, and a slide switch can be used to physically disconnect a tracker's battery.
+
+### Microcontroller Boards
+
+<table>
+    <tr>
+        <th>Board</th>
+        <th>Description</th>
+        <th>Obtaining</th>
+    </tr>
+    <tr>
+        <td>SuperMini nRF52840</td>
+        <td>It is a clone of the <strong>nice!nano</strong> board.</td>
+        <td>Cheapest option overall.<br/>
+         It is available on AliExpress with <code>compatible with nice!nano</code> or <code>Pro Micro</code> branding.</td>
+    </tr>
+    <tr>
+        <td>Seeed Studio XIAO nRF52840</td>
+        <td>Compact board.</td>
+        <td>Available on <strong>Seeed Studio</strong>.</td>
+    </tr>
+</table>
+
+### Inertial Measurement Units
+ - BMI270
+ - ICM-42688-P
+ - ICM-42688-V
+ - ICM-45686
+ - ISM330BX
+ - ISM330DHCX
+ - LSM6DSO
+ - LSM6DSR
+ - LSM6DSV
+ - LSM6DSV16B
+
+### Magnetometers
+ - AK09940
+ - BMM150[^sensorsNote]
+ - BMM350[^sensorsNote]
+ - IIS2MDC
+ - IST8306
+ - IST8308
+ - LIS2MDL
+ - LIS3MDL[^sensorsNote]
+ - MMC5983MA
```

Comment:
This is a nice list and all, but there's nothing telling people which of these are good, why they are good, their price range, etc etc. I get it that for mags this isn't very well known yet, but at least IMUs could be ""ranked"" for a lack of a better word